README.md

```
@@ -4,12 +4,305 @@ Our app is a centralized platform designed to bridge the gap between information
+#### Comments / Discussions
+- `GET /comments/:articleId`
+- `POST /comments`
+
+Full API details available in: ```backend/README_API.md'''
+
+
+---
+
+## Setup Instructions
+
+#### 1. Clone the Repository
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)